l/libiio/stone.yaml

```diff
@@ -0,0 +1,28 @@
+#
+# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name: libiio
+version: v0.26
+release: 1
+homepage: https://github.com/analogdevicesinc/libiio
+upstreams:
+  - https://github.com/analogdevicesinc/libiio/archive/refs/tags/v0.26.tar.gz: fb445fb860ef1248759f45d4273a4eff360534480ec87af64c6b8db3b99be7e5
+summary: A cross platform library for interfacing with local and remote Linux IIO devices
+description: |
+  libiio is used to interface to the Linux Industrial Input/Output (IIO) Subsystem.
+builddeps:
+  - binary(bison)
+  - binary(flex)
+  - libaio-devel
+  - pkgconfig(avahi-glib)
+  - pkgconfig(libusb-1.0)
+  - pkgconfig(libxml-2.0)
```

Comment:
`pkgconfig` dependencies are ordered at the top of the list

Reply:
Are you sure? when I looked as systemd for example you had the order binary, regular package names and last pkgconfig. kbd is similar ordered.